### PR TITLE
Clients invoice in installments

### PR DESCRIPTION
**Pull request by kanuj365**
Targets #2729

### Description
- In this feature we provide a functionality that Clients can pay partial payments or has paid the invoice in installments.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by GauravGusain98** [comment]
### Feedbacks
1. The partially paid option is not available 
image

2. I have not paid anything but the remaining amount is 0 by default
image

3. Also on adding the amount in the received amount field the remaining amount should change. This can be easily done  using vuejs
image

**Comment by kanuj365** [comment]
image
if user paid in installments then every amount after installments amount can see.

**Comment by kanuj365** [comment]
image

File: Modules/Invoice/Config/config.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
     'status' => [
         'sent' => 'Sent',
         'paid' => 'Paid',
+        'partially' => 'Partially Paid',
```
Comment (GauravGusain98) [inline Modules/Invoice/Config/config.php:8]:
```suggestion
        'partially_paid' => 'Partially Paid',
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
### Testing feedbacks
1. Partially paid Status should also be shown here
    image

2. The status should come as `Partially Paid` in orange color
image


3. The amount related columns should be encrypted
    image

I think I'll update the requirement of this task a bit. So please give me some time to do that. Till then please fix the given feedbacks

**Comment by GauravGusain98** [comment]
### Testing Feedbacks
- [ ] I am getting this error. Mostly looks like php issue. 
    image

- [ ] The status field is not coming up correct
    image

- [ ] The partially paid filter is not showing correct data
    image

- [ ] If an invoice is partially paid then we should not show overdue status, instead show partially paid status
    image

**Comment by GauravGusain98** [comment]
### Additonal requirements
- [ ] We need to make some changes in the current table [schema](https://docs.google.com/presentation/d/1djNuUlhABvhZieWBax39jshAGEGsComSzvzMKtMQv6w/edit#slide=id.g1c14b30e04f_0_134) use this file to update the `invoice_details` table
- [ ] If any invoice payment details is capture then make an entry for that payment in this table
- [ ] For partially paid invoices. The amount paid till now should be calculated by adding the amount paid information of all the entries for an invoice in this table
- [ ] If there is any previous payment details available then show the details like shown in below image
image

**Comment by GauravGusain98** [comment]
### Testing feedbacks
1. Overdue is still showing
2. Remaining amount is only decreasing. Its not resetting if we change the paid amount to 0
3. Rename the button text to Payment History
4. Make all the field default values as empty
5. Make details in accordion
    image
6. Store paid payment details also in new table

**Pull request closed without merging**